connman/gweb/gweb.c

```diff
@@ -1380,9 +1413,18 @@ static guint do_request(GWeb *web, const char *url,
 		session->address_action = g_timeout_add(0, already_resolved,
 							session);
 	} else {
+		// TODO make to use return value
+		int err = call_dns_route_func(session, true);
+		if (err)
```

Comment:
why not: if (err && err != -EALREADY)

Reply:
Yeah, that would work :)

Reply:
Changed with [a4e3c22](https://github.com/sailfishos/connman/pull/55/commits/a4e3c2202bab7ec34a107eac78bab9c12b6567ff)